src/assets/content/angular-calendar.json

```diff
@@ -78,7 +78,16 @@
       }
     },
     {
-      "day": 7
+      "day": 7,
+      "content": {
```

Comment:
Hi @LayZeeDK,
The schema was slight changed. Now we can display more than one guest per day, so it should be:
```json
{
      "day": 7,
      "contents": [
        {
          "author": {...},
          "videoSrc": "PLACEHOLDER"
        }
      ]
    },
```.

Thank you :)